SMLHelper/Assets/ModPrefab.cs

```diff
@@ -22,6 +22,7 @@ internal static void Add(ModPrefab prefab)
         }
 
         internal static IEnumerable<ModPrefab> Prefabs => PreFabsList;
+        internal static int PrefabCount = PreFabsList.Count;
```

Comment:
I believe you intended this to be a get-only property instead of a field that's written to once and  never updated.
```CS
internal static int PrefabCount => PreFabsList.Count;
```

Reply:
Woops yup

Reply:
Turns out this was not required as the prefab cache is wiped and rewritten every time the game checks for a prefab so our code needs to run at that time too. Kinda makes a cache useless but hey UWE